src/main/java/com/datasonnet/plugins/DefaultYamlFormatPlugin.java

```diff
@@ -0,0 +1,145 @@
+package com.datasonnet.plugins;
+
+/*-
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.datasonnet.document.DefaultDocument;
+import com.datasonnet.document.Document;
+import com.datasonnet.document.MediaType;
+import com.datasonnet.document.MediaTypes;
+import com.datasonnet.spi.PluginException;
+import com.datasonnet.spi.ujsonUtils;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.dataformat.yaml.YAMLParser;
+import ujson.Value;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.util.List;
+
+public class DefaultYamlFormatPlugin extends BaseJacksonDataFormatPlugin {
+
+    public static final String DS_PARAM_YAML_HEADER = "removehead";
```

Comment:
Is header the official / accurate term in yaml?

Reply:
Not really, it doesnt have a name for it: https://yaml.org/spec/1.1/#id857577

Reply:
If you click the three dashes, they have a name in that section, Document Boundary Markers